tensorflow/python/keras/applications/resnet.py

```diff
@@ -538,13 +538,16 @@ def decode_predictions(preds, top=5):
 
 DOC = """
 
-  Reference paper:
-  - [Deep Residual Learning for Image Recognition]
-  (https://arxiv.org/abs/1512.03385) (CVPR 2015)
+  Reference:
+  - [Deep Residual Learning for Image Recognition](
+      https://arxiv.org/abs/1512.03385) (CVPR 2015)
 
   Optionally loads weights pre-trained on ImageNet.
   Note that the data format convention used by the model is
   the one specified in your Keras config at `~/.keras/keras.json`.
+  
+  Caution: Be sure to properly pre-process your inputs to the application.
+  Please see `applications.resnet.preprocess_input` for an example.
```

Comment:
Good work, maybe add a link here?

https://www.tensorflow.org/api_docs/python/tf/keras/applications/resnet/preprocess_input

Reply:
If we are going to add the link here, it should be added to all other nets too, I think.
Additionally I noticed that densenet, nasnet are missing caution too (and reference is not as in https://github.com/tensorflow/tensorflow/commit/71964116c54e3f1ad9686b9fb4987d526a15d8a7).
I can convert this PR to update all other networks in the folder?

Reply:
Revisiting this a few days later I remembered that this caution will actually generate proper link on TensorFlow site (see [here](https://www.tensorflow.org/api_docs/python/tf/keras/applications/ResNet101V2) for an example), so I'll mark this as resolved. Still need to updated other nets with the same change though.